components/SharingSettingsPanel.test.tsx

```
@@ -0,0 +1,299 @@
+import { gql } from "@apollo/client";
+
+describe("SharingSettingsPanel Component", () => {
+  describe("Component Props Interface", () => {
+    it("should define correct component structure", () => {
+      interface SharingSettingsPanelProps {
+        // No props required for this component
+      }
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)